handler.py

```diff
@@ -0,0 +1,30 @@
+import json
+import datetime
+import logging
+
+from util.s3_helper import send_to_s3
+from lib.target import Target
+from scanners.httpobs_scanner import HTTPObservatoryScanner
+from util.host_picker import Randomizer
+from util.response import Response
```

Comment:
Are you using Response anywhere?  The goal of Response was basically to "securitify" responses, but  in the handler I'm not seeing explicit responses denoted.

Reply:
If we're not using it, I think we should remove it.  If we intend to use it, we should be explicit about what the handler  responds with when a scan is tasked and returned.

Reply:
Yeah, this branch does not use it yet. Will include in other branches (and eventually in master).